ingenialink/ethercat/servo.py

```diff
@@ -320,6 +324,75 @@ def set_pdo_watchdog_time(self, timeout: float) -> None:
             self.ETHERCAT_PDO_WATCHDOG, self.SECONDS_TO_MS_CONVERSION_FACTOR * timeout
         )
 
+    def _read_esc_eeprom(
+        self,
+        address: int,
+        length: int = DEFAULT_EEPROM_READ_BYTES_LENGTH,
+        timeout: int = DEFAULT_EEPROM_OPERATION_TIMEOUT_uS,
+    ) -> bytes:
+        """Read from the ESC EEPROM.
+
+        Args:
+            address: EEPROM address to be read.
+            length: Length of data to be read. By default, 2 bytes are read.
+            timeout: Operation timeout (microseconds). By default, 200.000 us.
+
+        Returns:
+            EEPROM data. The read data.
+
+        Raises:
+            ValueError: If the length to be read has an invalid value.
+
+        """
+        if length < 1:
+            raise ValueError("The minimum length is 1 byte.")
+        data = bytes()
+        start_address = address
```

Comment:
Is it necessary to create a new variable? In fact start_address increases on the loop so it's name loses sense

Reply:
Done